### DISCUSSION TITLE
Laser ne grave plus. Ortur Laser Master 2

### DISCUSSION THREAD

**Discussion by didiergrazy** [arkypita/LaserGRBL#1159] [Q&A]
Bonjour je possède une laser Ortur master2 je l'ai depuis un mois ,tout marchais très bien et ce matin je l'allume mes déplacements ce font bien mais le laser ne brule plus??? Alors que je n'ai rien changer dans mes réglages LaserGrbl . Une personne aurait il u ce problème. Merci a tous.

**Comment by arkypita** [comment]
LaserGRBL is a free software downloadable from the internet, not tied to a specific product.
I suggest you to contact the specialized assistance of your engraver.

For ORTUR brand:
- [Ortur user group (facebook community)](https://www.facebook.com/groups/orturusergroup)
- [Ortur customer support](https://ortur.tech/support/)

Best regards
Arkypita, developer of LaserGRBL
